### PR TITLE
Minor fix: eventFilter drop on minimized; comments lost sync after GUI rest

### PR DESCRIPTION
**Pull request by davidlatwe**
This PR will provide two fix:
1. eventFilter get removed when main window minimized.
    > did not found this bug on previous PR, sorry :(
2. Comment did not restore back to host after rest, here's the scenario:
    1. Artists input comments to QML comment box
    2. Validation failed or some other cases that make artist to **press `reset` botton**
    3. Comments remain in QML comment box and **did not make any new input**
    4. Published, but the comment did not collect to database

### CODE REVIEW

File: pyblish_qml/control.py [hunk 1]
```diff
@@ -617,6 +617,11 @@ def echo(self, data):
         """Append `data` to result model"""
         self.data["models"]["result"].add_item(data)
 
+    def comment_sync(self, comment):
+        """Update comments to host and notify subscribers"""
+        self.host.update(key="comment", value=comment)
+        self.host.emit("commented", comment=comment)
+
```
Comment (davidlatwe) [inline pyblish_qml/control.py]: I was trying this kind of simple (lazy) fix.
Dose this work for you ? @mottosso

File: pyblish_qml/ipc/server.py [hunk 2]
```diff
@@ -112,8 +112,9 @@ def show(self, settings=None):
 
     def hide(self):
         """Hide the GUI"""
-        self._dispatch("hide")
-        return self.vessel.hide()
+        success = self._dispatch("hide")
+        self.vessel.hide()
+        return success
```
Comment (davidlatwe) [inline pyblish_qml/ipc/server.py]: `eventFilter` will get removed if `proxy._dispatch` returns `False`/`None`, which suggest that subprocee connection has closed.
It's obvious that returning the result of `self.vessel.hide()` is incorrect.

**Comment by davidlatwe** [comment]
Much appreciated !
Merge will be made right after I got back to my workplace and bump the version.
Thanks again @mottosso  :)

**Comment by davidlatwe** [comment]
Sorry for the spamming, didn't know that `git rebase` will break GitHub code review link, so I did a reset and re-bump the version with conflict resolved.

Merging now :)

**Comment by mottosso** [comment]
Thanks @davidlatwe :)

**Pull request merged**